_posts/2019-07-12-barcelona-announcement.md

```diff
@@ -0,0 +1,82 @@
+
+We took into account the feedback we received from previous editions and we believe this can work well for most of you. Indeed, this will allow us to have more talks and workshops and reduce the intensity of a monothematic format day.
+
+## Venue
+
+Rustfest Barcelona 2019 is going to be 
+in the [Vèrtex building at Campus Nord](https://www.upc.edu/campusnord/ca/espais/auditori) belonging to [UPC - Universitat Politècnica de Catalunya](https://www.upc.edu/en).
+
+![Vèrtex Auditorium](/assets/posts/barcelona-vx_auditori.jpg)
+
+We'll be using the two level **Vèrtex auditorium** with capacity for 477 people and access for people with reduced mobility at any of the two levels which we think will make a great venue for the amazing talks we'll be hosting this year. ;D
+
+The workshops and the *Impl Days* will take place in some of the [university's classrooms](https://www.upc.edu/campusnord/ca/espais/aules-vertex) which are fully equipped to host workshops of almost any kind.
+
+We will have also available a chilling area for resting between sessions.
+
+The venue is wheelchair accessible over the lateral street and can be reached easily by public transport. For more information about accesss for people with a disability in Barcelona you can take a look at [this link](https://www.barcelona-tourist-guide.com/en/transport/disabled/access-barcelona-disabled.html).
+
+The best options to get there, if you come from Sants Estació, are the [Av. Pedralbes - Cavallers bus station](https://goo.gl/maps/QeNjywZ749Njur8S8) and the [Av d’Esplugues - Av Pearson bus station](https://www.google.com/url?q=https://www.google.com/maps/dir/Sants%2BEstaci%25C3%25B3,%2BBarcelona/Av%2Bd%27Esplugues%2B-%2BAv%2BPearson,%2B08034%2BBarcelona/@41.386477,2.1192745,15z/data%3D!4m15!4m14!1m5!1m1!1s0x12a4987e1c029355:0xb19606f1fb16429e!2m2!1d2.1409712!2d41.3816151!1m5!1m1!1s0x12a4984e1e97de21:0x992b87b41d670e10!2m2!1d2.111625!2d41.3921394!3e3!5i3!5m1!1e2&sa=D&ust=1562873148055000&usg=AFQjCNE1co3JRi_anes-tAbTSK8FFGFX4g); also there’s a subway option to [Palau Reial station](https://www.google.com/url?q=https://goo.gl/maps/4u1dJbpRUHRrrmxE8&sa=D&ust=1562873148056000&usg=AFQjCNHwY0Vd1hK9hNsxd68fmNxsLdHs1A) if you want a quicker route that will require some walk.
+
+Anyway, don't worry, we will update our website soon with plenty of information on this regard, so stay tuned! ;P
+
+## Catering
+
+We will provide breakfast, coffee, drinks and lunch the first two conference days so you don't have to worry about looking for a restaurant.
+
+We'll also have vegan and vegetarian options and, if you have other dietary needs, please just contact us and we'll be happy to help you.
+
+For the *Impl Days*, it will be up to you to decide where to eat. But don't worry, we'll gather information about some good restaurants in the area and we will fill our website with it so you can experience the best of our great gastronomy.
```

Comment:
I would leave out the "don't worry".
```suggestion
For the *Impl Days*, it will be up to you to decide where to eat. We'll gather information about some good restaurants in the area and provide a list of options on our website, so you can experience the best of Barcelona's great gastronomy.
```